samples/subsys/zbus/benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(benchmark)
+
+zephyr_compile_definitions(
+        -DBM_MESSAGE_SIZE=${MSG_SIZE}U -DBM_ONE_TO=${ONE_TO}LLU -DBM_ASYNC=${ASYNC}U
```

Comment:
These should really be sample specific Kconfig options (`ZBUS_SAMPLE_BENCHMARK_MSG_SIZE` etc...). Then you won't have to use `extra_args` but you can use `extra_configs` instead.